docs/contrib/Logger.rst

```diff
@@ -1,29 +1,94 @@
 Using the logging infrastructure
 ================================
 
-In order to ensure consistent messaging from the openroad application we
-have adopted spdlog as our logging infrastructure. We have a thin
-wrapper on top for extensibility. Whenever a message needs to be issued
-you will use one of the logging functions in the ‘ord’ namespace.
+OpenROAD uses spdlog as part of logging infrastructure in order to ensure a clear, consistent messaging and complete messaging interface. 
+A wrapper formats the prefix in the reccommended messaging style and limit. 
+A message format is as follows:
+<tool id> <message id> : <Message body>.
+
+For Example,
+[INFO ODB-0127] Reading DEF file: ./results/asap7/aes/base/4_cts.def
```

Comment:
The text on lines 7 and 10 should be in some kind of block so they are differentiated when formatted.

Either wrap them with double grave accent or create a code-block.

Example [here for grave accent](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/docs/contrib/DatabaseMath.rst) where it references C/C++ types.

Example [here for code-blocks](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/docs/contrib/CodingPractices.rst#practice-6).